scripts/index.js

```diff
@@ -0,0 +1,110 @@
+const state = {
+    currentTemp: 65,
+    currentSky: "Sunny",
+    cityNameDefault: "Issaquah"
+};
+
+const updateTemp = () => {
+    const tempValueContainer = document.querySelector("#tempValue");
+    tempValueContainer.textContent = state.currentTemp;
+};
+
+const increaseTemp = () => {
+    state.currentTemp += 1;
+    updateTemp();
+    updateTempColor();
+    updateLandscape();
+};
+
+const decreaseTemp = () => {
+    state.currentTemp -= 1;
+    updateTemp();
+    updateTempColor();
+    updateLandscape();
+};
+
+const updateTempColor = () => {
+    const tempValueContainer = document.querySelector("#tempValue")
+    let color = "yellow";
+    if (state.currentTemp >= 80) {
+        color = "red";
+    } else if (state.currentTemp >= 70) {
+        color = "orange";
+    } else if (state.currentTemp >= 60) {
+        color = "yellow";
+    } else if (state.currentTemp >= 50) {
+        color = "green";
+    } else {
+        color = "blue";
+    };
+    tempValueContainer.classList = color;
+};
+
+// potentially combine this in with updateTempColor since ranges of temps are the same
```

Comment:
I'm glad you noticed this.  :smiley:

It's not a big deal to repeat this but definitely could make sense to combine them, especially if the logic got more complicated.